tests/mocha/workspace_test.js

```diff
@@ -247,7 +247,9 @@ function testAWorkspace() {
 
   suite('addTopBlock', function() {
     setup(function() {
-      this.targetWorkspace = new Blockly.Workspace();
+      this.targetWorkspace = this.workspace.rendered ?
```

Comment:
When would the workspace not be a workspace svg, if it's a flyout?

Reply:
I suppose in headless it might not make sense to test flyout. Is that the case?
I could instead:

- Move this to WorkspaceSvg tests 
- OR do something like:

```
this.targetWorkspace = this.workspace.rendered ? this.workspace : new Blockly.Workspace();
if (this.workspace.rendered) {
  this.workspace.isFlyout = true;
  this.workspace.targetWorkspace = this.targetWorkspace;
}
```

Reply:
Moving it to workspace_svg sounds most reasonable to me.

Reply:
Done.